### DISCUSSION TITLE
Allow adding new blocksets while access to settings is restricted

### DISCUSSION THREAD

**Discussion by sirati** [proginosko/LeechBlockNG#349] [Ideas]
I want to add a new blockset while I have my access to settings restricted.

Basically I do not have enough self control and restricted access to settings to a short time period at which I am usually asleep. This works great in stopping me from messing with the settings, however it unfortunately also made it impossible to add new blocksets outside this time. 

I hope this is not too ridiculous

**Comment by proginosko** [comment]
How would that work in practice? What would the interface look like?

**Comment by sirati** [comment]
Here is a possible light implementation that does what i want:

1. blocksets have a new hidden setting "isFinalised"
2. Add two new buttons to the "settings are currently blocked screen": "add new blockset", "edit unfinalised blocksets"
3. both will take you the setting screen but only the tabs are shown of blocksets that have isFinalised = false. Clicking "add new blockset" will also increase the amount of blocksets by one and ensure isFinalised is set to false for the new blockset
4. These blocksets are able to be edited just normal. but they show a "finalise" button upon which isFinalised is set to true and their tab is removed

if this is would only be accessible through an optional setting:
1.  if this setting is off all blocksets created will start with isFinalised is set to true (upon restricting access to settings they all will not be accessible as expected)
